internal/resources/fabric/precision_time/models.go

```diff
@@ -0,0 +1,308 @@
+package precision_time
+
+import (
+	"context"
+	"github.com/equinix/equinix-sdk-go/services/fabricv4"
+	fwtypes "github.com/equinix/terraform-provider-equinix/internal/framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+)
+
+type ResourceModel struct {
```

Comment:
IMO it's preferable to have a single, shared `Model` if the implementations are identical (these ones look the same to me but I might be missing a difference somewhere).